docs/our_work/ds278_Assurance_Research_Path.md

```diff
@@ -0,0 +1,29 @@
+---
+title: 'DS_278: Assurance Research Path'
+summary: 'Investigating new techniques to test and assure AI and machine learning solutions in the NHS'
+origin: 'NHS England'
+tags: ['NLP','LLM','GENAI',Assurance, Machine Learning, Classification, Computer Vision, Work in Progress]
+---
+
+The NHS AI Quality Community of Practice (AIQ CoP) is a network of testing and assurance professionals dedicated to supporting the assurance of healthcare AI. While not an assurance function itself, the AIQ CoP provides guidance, resources, and advice to assist colleagues and organisations in navigating the complexities of AI assurance. While the project sits within NHS England, the remit of the AIQCOP is the whole of the NHS. 
```

Comment:
be consisrent on how you capitalise the acronym so either AIQ CoP or AIQCOP every time